server/src/entity/AskedQuestion.ts

```diff
@@ -17,6 +17,12 @@ export class AskedQuestion {
   })
   questionId: string;
 
+  @Column('uuid', {
+    name: 'session_id',
```

Comment:
If we are talking about competition then this should be contest_id. A session is linked to the user registration process.